checks/cs50/2017/x/finance/checks.py

```diff
@@ -1,13 +1,305 @@
 import os
 import re
+import requests
 import sys
 
 sys.path.append(os.getcwd())
-from check50 import TestCase, Error, check
+from check50 import File, TestCase, Error, check
+from selenium import webdriver
 
 class Finance(TestCase):
+
+    BASE_PATH = "http://127.0.0.1:5000"
```

Comment:
For local testing in Docker, would `0.0.0.0` be more robust, since tends to bind to an internal IP? And `8080` for consistency with open IDE ports (and CS50 CLI's)?